### PR TITLE
Credentials & updated box version & fixed make warning

### LINKED ISSUE #52: Proper rendering of credentials
## Feature description
As a user of the module we want proper render of credentials, without using vault provider

## Why is it needed?
To make it easier to use the module.

## Suggestion(s)/solution(s) [Optional]
Done in Presto module: https://github.com/fredrikhgrelland/terraform-nomad-presto/blob/master/conf/nomad/presto.hcl#L327

## Definition of done
- Rendering credentials in `.hcl` file

## Checklist (after created issue)
- [x] Added label(s)
- [x] Added to project
- [x] Added to milestone

### LINKED ISSUE #57: Warnings running `make up`
## Current behaviour
Throws warnings when running targets from makefile
```
~/projects/terraform-nomad-presto(master) » make clean                                                                                                      m88614@SKE-DC6KF-MD6T
Makefile:77: warning: overriding commands for target `status'
Makefile:60: warning: ignoring old commands for target `status'
```
image

## Expected behaviour
No warnings

## How to reproduce?
Run `make clean` or `make up`

## Suggestion(s)/solution(s) [Optional]
Will look at why it happens tomorrow. Making this now so that I don't forget

## Checklist (after created issue)
- [x] Added label(s)
- [x] Added to project
- [x] Added to milestone

### LINKED ISSUE #66: Turn `ON` all optional features in example
## What is the issue?
Lacking optional features in tests:
- vault-provided credentials

## Checklist (after created issue)
- [x] Added label(s)
- [x] Added to project
- [x] Added to milestone

### LINKED ISSUE #69: Upgrade to 0.7.0
## What is the issue?
Upgrade and test v0.7.0 of the hashistack box. 
https://app.vagrantup.com/fredrikhgrelland/boxes/hashistack/versions/0.7.0

## Suggestion(s)/solution(s) [Optional]
Change `Vagrantfile.default` versioning to fetch newest box between 0.7 and 0.8.

## Checklist (after created issue)
- [x] Added label(s)
- [x] Added to project
- [x] Added to milestone

### PR DESCRIPTION
**Pull request by claesgill**
## Closes/fixes/resolves issue(s)?
Closes #69 
Closes #52 
Closes #57
Closes #66 

## What was added/changed/fixed?
- Rewrote some variables regarding credentials
- Updated readme
- Udated box version
- Fixed make warning

## Checklist (after created PR)
- [x] Added reviewers
- [x] Added assignee(s)
- [x] Added label(s)
- [x] Added to project
- [x] Added to milestone

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,10 +3,16 @@
 ## [0.3.0 UNRELEASED]
 
 ### Added
-
 - Json flatten VIEW example #65
 - Added CPU as user defined variable #67
 
+### Changed
+- Variables regarding credentials/secrets & updated documentation #52
+- Updated box version to 0.7.0 #69
```
Comment (pdmthorsrud) [inline CHANGELOG.md:11]:
```suggestion
- Updated box version to 0.7.x #69
```

File: README.md [hunk 2]
```diff
@@ -115,7 +187,60 @@ The following intentions are required. In the examples, intentions are created i
 ## Outputs
 | Name | Description | Type |
 |------|-------------|------|
-| presto\_service\_name | Presto service name | string |
+| presto_service_name | Presto service name | string |
+
+## Secrets & credentials
+When using the `mode = "cluster`, you can set your secrets in two ways, either manually or upload secrets to Vault.
```
Comment (pdmthorsrud) [inline README.md:193]: We should potentially add an issue to make the same available in the standalone version

File: CHANGELOG.md (hunk 1 above)
Comment (zhenik) [inline CHANGELOG.md:11]:
```suggestion
- Updated box version to 0.7.1 #69
```

File: README.md (hunk 2 above)
Comment (zhenik) [inline README.md:193]:
```suggestion
When using the `mode = "cluster"`, you can set your secrets in two ways, either manually or upload secrets to Vault.
```

**Review by zhenik — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (pdmthorsrud) [reply README.md:193]: Nvm, I'll do that in #61

File: Makefile [hunk 3]
```diff
@@ -73,8 +73,6 @@ remove-tmp:
 
 clean: destroy-box remove-tmp
 
-status:
-	vagrant global-status
```
[the remark this answers is not in the document]
Comment (claesgill) [reply Makefile:77]: There was two instances of this cmd. Still exist in line 59

**Review by pdmthorsrud — APPROVED** [review]

**Review by zhenik — APPROVED** [review]

**Review by pdmthorsrud — APPROVED** [review]

**Review by zhenik — APPROVED** [review]

**Pull request merged**